LondonBritishLibrary/orient/BLorient615.xml

```diff
@@ -344,10 +348,12 @@ of emperor <persName ref="PRS2377Bakaffa">Bakāffā</persName>.</desc>
         <revisionDesc>
             <change who="MKr" when="2021-07-19">Created entity</change>
             <change when="2022-06-09" who="DR">Added layout.</change>
+            <change when="2024-12-19" who="MV">Added new IDs</change>
         </revisionDesc>
     </teiHeader>
     <facsimile>
         <graphic url="http://www.bl.uk/manuscripts/Viewer.aspx?ref=or_615_f001r"/>
+        <!-- This URL is currently not working (MV 2024-12-19) -->
```

Comment:
and probably never will, the new URIs have a different structure, see eg Or 614 now at https://iiif.bl.uk/uv/#?manifest=https://bl.digirati.io/iiif/ark:/81055/vdc_100054870112.0x000001